.changeset/hip-toys-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/node': patch
+---
+
+Apply polyfills before awaiting the env module
```

Comment:
Can you add a better description for this change? I really don't understand what it is

Reply:
Done, thanks!